.github/workflows/perf-testing-setup.yml

```diff
@@ -24,10 +23,10 @@ jobs:
       uses: actions/setup-python@v2
       with:
         python-version: ${{ env.PYTHON_VERSION }}
-    - name: Set up Dotnet 3.1.x
+    - name: Set up Dotnet 2.x
       uses: actions/setup-dotnet@v1
       with:
-        dotnet-version: '3.1.x'
+        dotnet-version: '2.2.207'
```

Comment:
Version 3.1 doesn't build the host. This was failing.

Reply:
Yeah, I think to enable 3.1 dotnet build, we also need to update the host link in setup.py https://github.com/Azure/azure-functions-python-worker/blob/dev/setup.py#L21.

But this host update should be placed in a separate PR, don't need to worry about it for now.